paddlenlp/transformers/gpt/modeling.py

```diff
@@ -651,6 +702,58 @@ def forward(self,
                 attention_mask=None,
                 use_cache=False,
                 cache=None):
+        r'''
+        The GPTModel forward method, overrides the `__call__()` special method.
+
+        Args:
+            input_ids (Tensor):
+                Indices of input sequence tokens in the vocabulary. They are
+                numerical representations of tokens that build the input sequence.
+                Its data type should be `int64` and it has a shape of [batch_size, sequence_length].
+            position_ids(Tensor, optional):
+                Indices of positions of each input sequence tokens in the position embeddings. Selected in the range ``[0,
+                max_position_embeddings - 1]``.
+                Shape as `(batch_size, num_tokens)` and dtype as int64. Defaults to `None`.
+            attention_mask (Tensor, optional):
```

Comment:
We are using INF mask actually.

-1e-9 for masked position. 0 for unmasked.
https://github.com/PaddlePaddle/PaddleNLP/blob/develop/examples/language_model/gpt/dataset.py#L364

Reply:
done